eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Compiler/SlugConverterConfigurationPass.php

```diff
@@ -61,6 +61,18 @@ public function process(ContainerBuilder $container)
                 $mergedConfiguration['transformation'],
                 implode(', ', array_keys($mergedConfiguration['transformationGroups']))
             ));
+        } elseif (empty($mergedConfiguration['transformation'])) {
+            $container->log(
+                $this,
+                sprintf(
+                    'Relying on default url_alias.slug_converter.transformation setting ("%s") is deprecated and might change in the next major. Set it explicitly to one of the following: %s',
+                    SlugConverter::DEFAULT_CONFIGURATION['transformation'],
+                    implode(
+                        ', ',
+                        array_keys(SlugConverter::DEFAULT_CONFIGURATION['transformationGroups'])
```

Comment:
Should that help message not relay on $mergedConfiguration instead of default one? Or we assume that if someone provides additional groups he have one that already set?

Reply:
@ViniTou I rather assumed that if transformation is not set, we can safely rely on default transformationGroups. 

Otherwise we'd need a code to check if `$mergedConfiguration` has `transformationGroups` and if not, use default ones. That said, with PHP 7.x `??` operator I think that wouldn't complicate solution too much, so I can change it if you want :)

Reply:
`$mergedConfiguration` should always have `transformationGroups`. I don't have strong opinion about that anyway, just curious :)

Reply:
> `$mergedConfiguration` should always have `transformationGroups`.

@ViniTou debugging doesn't seem to confirm that. If I don't set this via semantic conf (e.g. on meta) or via overriding service, key doesn't exist. 

However I missed a couple of lines above `$transformationGroups` variable which already has everything I needed. Fixed in ed1ff57.

On the bright side, I finally figured out where `E_*DEPRECATED` errors go during compiling container - there's the `var/cache/dev/appDevDebugProjectContainerDeprecations.log` (or similar depending on ENV and DEBUG settings). 

So I also changed code to trigger deprecation error instead of just logging. That was the intention anyway.